mergeable/readme.md

```diff
@@ -1,16 +1,14 @@
-Nodejs app to check the open PRs of a github repository and ping slack if they're deemed unmergable.
+`Nodejs` app written in `TypeScript` to check the open PRs of a github repository and ping slack if they're deemed unmergable.
 
 
 ## Usage
 
-Create a secrets.json with the following structure:
+Create a secrets.json in the root with the following structure:
```

Comment:
if instead of a json you have a ts file, you will have compile time safety (you actually get to know which keys the options have)